modules/navier_stokes/doc/content/source/auxkernels/ReynoldsNumberFunctorAux.md

```diff
@@ -0,0 +1,31 @@
+# ReynoldsNumberFunctorAux
+
+!syntax description /AuxKernels/ReynoldsNumberFunctorAux
+
+## Overview
+
+The `ReynoldsNumberFunctorAux` object computes the element/cell Reynolds number
+given by
+
+\begin{equation}
+Re = \frac{\rho h u}{\mu}
+\end{equation}
+
+where $u$ is the norm of the velocity, e.g. the speed, $h$ the largest dimension
+of the element (computed using `_current_elem->hmax()`), $\rho$ is the density,
```

Comment:
technically should probably be h in the direction of velocity
and wouldnt hmin be more limiting?

Reply:
L is in the numerator so if you're worried about possibly needing turbulence treatment or stabilization then the paranoid estimate is with the largest cell/element L

Reply:
oh yes my bad